docs/readmes/proposals/p021_mme_migrate_to_c++.md

```diff
@@ -0,0 +1,143 @@
+---
+id: p021_mme_migrate_to_c++
+title: C++ Migration
+hide_title: true
+---
+
+
+# Proposal: C++ Migration
```

Comment:
since this proposal is centered around studying state representation and serialization performance, perhaps we should rename it as such?

Reply:
I would say it is the first task to address, since it may add some serious constraints to the data model. Sure it is a big subject, that seems to shadow the C++ migration.